circleci/docker-publish

```diff
@@ -75,6 +75,8 @@ check_ecr_vars
 install_awscli
 
 # Some Dockerfiles for private repos depend on public images (and vice versa) in us-west-1
+echo "If necessary, add the ECR_BUILD_ID and ECR_BUILD_SECRET env vars to circle manually.
+They can be found in init-service as CI_ECR_XXX_KEY and CI_ECR_XXX_SECRET"
```

Comment:
Is this correct bash? You don't need a backslash here?

Reply:
Pretty sure a backslash isn't necessary since it's inside quotes. But I can just switch it to a separate `echo` for each line.